testsuite/scripts/src/test/java/org/wildfly/scripts/test/StandaloneScriptTestCase.java

```diff
@@ -94,6 +94,7 @@ public StandaloneScriptTestCase() {
         final Collection<Object> result = new ArrayList<>(2);
         result.add(Collections.emptyMap());
         result.add(Collections.singletonMap("GC_LOG", "true"));
+        result.add(Collections.singletonMap("MODULE_OPTS", "-javaagent:logging-agent-tests.jar"));
```

Comment:
When this is set we should check the message is logged. Also maybe we should test this with GC_LOG set too.